rrbot_control/launch/rrbot_control.launch

```diff
@@ -15,4 +15,7 @@
     <remap from="/joint_states" to="/rrbot/joint_states" />
   </node>
 
+  <!-- Show in Rviz   -->
+  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find rrbot_description)/launch/rrbot.rviz"/>
```

Comment:
I don't having Rviz launch from rrbot_control, it does not seem intuitive and I think it should stay separate. Could you remove this but perhaps document launching this elsewhere.